independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -449,9 +447,6 @@ protected Class<?> findClass(String moduleName, String name) {
 
     @Override
     public Class<?> loadClass(String name) throws ClassNotFoundException {
-        for (ClassLoaderEventListener l : classLoaderEventListeners) {
-            l.loadClass(name, this.name);
-        }
         return loadClass(name, false);
```

Comment:
Nice catch. Strange that we didn't notice as this is the one thing that integration tests actually use :)
I'd have framed it as a bugfix though rather than a micro-optimisation.